docs/content/developers/building-contributing.md

```diff
@@ -135,6 +135,38 @@ To manually push using GitHub Actions,
 * Click “Run workflow” in the blue band near the top.
 * Choose the branch, usually `master`. Include a tag for the pushed image and GitHub will do all the work.
 
+## Instrumentation
+
+The instrumentation implementation is generated using the [Ampli Codegen](https://www.docs.developers.amplitude.com/data/sdks/go/ampli/).
+
+To synchronize the implementation with the latest changes at Amplitude, the CLI
+tool has to be installed locally:
+
+```bash
+npm install -g @amplitude/ampli
+```
+
+Make changes to the event definition using the GUI at <https://data.amplitude.com/ddev/DDEV>:
+
+* create a new branch
+* create or change events and properties
+* save changes to the new branch
+* update the implementation with `ampli checkout <branch name>`
+* make changes to the code
+
+Make sure new properties are not breaking, add it as optional properties though.
```

Comment:
I’m only following along theoretically, but I’m confused by this sentence. (The plural “new properties” followed by singular “it” could probably be corrected regardless.) Are you saying to make sure that each new property doesn’t break anything, but to make it optional for safety?

Reply:
It looks like this can be removed. The API does not check the properties to be there, it's for the code generation only which I did not know at the point where I wrote it.